### PR TITLE
:memo: Showcase on website/documentation as reward for contributor

### PR DESCRIPTION
**Pull request by Okazari**
I wanted to add a mention about showcase contributors on your website/documentation. Tell me if you feel that it isn't necessary. If you want i can find another example so that it will less feel as a rythm.js placement :open_mouth: 

Also tell me if you think this precision isn't necessary :)

### CODE REVIEW

File: README-fr.md [hunk 1]
```diff
@@ -209,7 +209,7 @@ Une langue est manquante ? Vous pensez qu'il est possible d'améliorer ce cheat
...
+> Certains projets open-source comme [gatsby](https://github.com/gatsbyjs/gatsby) récompensent leurs contributeurs avec des goodies. Si vous n'en n'avez pas les moyens, faites une publication (sur twitter ou d'autres plateformes) qui mentionne la contribution et son auteur afin de le remercier (ex: [Post sur twitter](https://twitter.com/FranckAbgrall/status/1139470547492978688)). Vous pouvez aussi ouvrir une section `Contributors` sur votre README afin d'afficher publiquement votre gratitude envers vos contributeurs (ex: [Section contributeurs de vuepress](https://github.com/vuejs/vuepress#code-contributors)). Vous pouvez également les mettre en avant sur le site/la documentation de votre projet (ex: [Page de démo de Rythm.js](https://okazari.github.io/Rythm.js/))
```
Comment (kefranabg) [inline README-fr.md]:
```suggestion
> Certains projets open-source comme [gatsby](https://github.com/gatsbyjs/gatsby) récompensent leurs contributeurs avec des goodies. Si vous n'en n'avez pas les moyens, faites une publication (sur twitter ou d'autres plateformes) qui mentionne la contribution et son auteur afin de le remercier (ex: [Post sur twitter](https://twitter.com/FranckAbgrall/status/1139470547492978688)). Vous pouvez aussi ouvrir une section `Contributors` sur votre README afin d'afficher publiquement votre gratitude envers vos contributeurs (ex: [Section contributeurs de vuepress](https://github.com/vuejs/vuepress#code-contributors)) ou encore les mettre en avant sur le site/la documentation de votre projet (ex: [Page de démo de Rythm.js](https://okazari.github.io/Rythm.js/))
```

File: README.md [hunk 2]
```diff
@@ -210,7 +210,7 @@ A language is missing? Want to improve this cheat sheet? [Contributions are welc
...
-> Be nice with people that helped you! Some open-source projects like [gatsby](https://github.com/gatsbyjs/gatsby) reward contributors with goodies. If you can't afford that, do a public post (on twitter or other platforms) about the contribution and mention the author ([here is an example of public thanks](https://twitter.com/FranckAbgrall/status/1139470547492978688)). Open a `Contributors` section in your README to publicly thanks them (ex: [vuepress contributors section](https://github.com/vuejs/vuepress#code-contributors)).
+> Be nice with people that helped you! Some open-source projects like [gatsby](https://github.com/gatsbyjs/gatsby) reward contributors with goodies. If you can't afford that, do a public post (on twitter or other platforms) about the contribution and mention the author ([here is an example of public thanks](https://twitter.com/FranckAbgrall/status/1139470547492978688)). Open a `Contributors` section in your README to publicly thanks them (ex: [vuepress contributors section](https://github.com/vuejs/vuepress#code-contributors)). You can also showcase them on your project documentation/website (ex: [Rythm.js demo page](https://okazari.github.io/Rythm.js/))
```
Comment (kefranabg) [inline README.md]:
```suggestion
> Be nice with people that helped you! Some open-source projects like [gatsby](https://github.com/gatsbyjs/gatsby) reward contributors with goodies. If you can't afford that, do a public post (on twitter or other platforms) about the contribution and mention the author ([here is an example of public thanks](https://twitter.com/FranckAbgrall/status/1139470547492978688)). Open a `Contributors` section in your README to publicly thanks them (ex: [vuepress contributors section](https://github.com/vuejs/vuepress#code-contributors)) or showcase them on your project documentation/website (ex: [Rythm.js demo page](https://okazari.github.io/Rythm.js/))
```

**Comment by kefranabg** [comment]
@all-contributors please add Okazari for docs

**Comment by thomas-betous-cbp** [comment]
Great idea, but I think we should have more examples. Not only because it could see as a placement, but also because we should show different ways to show how it could be done.

I think the way rythm.js do it is nice, but we could show also https://conference-hall.io/speaker/contributors (unfortunatly need to be logged)

What do you think about that ? Do you have other examples ? Two more should be nice !

**Comment by tbetous** [comment]
Great idea, but I think we should have more examples. Not only because it could see as a placement, but also because we should show different ways to show how it could be done.

I think the way rythm.js do it is nice, but we could show also https://conference-hall.io/speaker/contributors (unfortunatly need to be logged)

What do you think about that ? Do you have other examples ? Two more should be nice !

**Comment by Okazari** [comment]
@tbetous I took the inspiration from @botpress project. When you install it, you have a backoffice board that showcase contributors in a fancy way :)
We could add a link to an uploaded image of the board ? (if they still do it)

File: README.md [hunk 3]
```diff
@@ -210,7 +210,7 @@ A language is missing? Want to improve this cheat sheet? [Contributions are welc
...
 <p>
 
-> Be nice with people that helped you! Some open-source projects like [gatsby](https://github.com/gatsbyjs/gatsby) reward contributors with goodies. If you can't afford that, do a public post (on twitter or other platforms) about the contribution and mention the author ([here is an example of public thanks](https://twitter.com/FranckAbgrall/status/1139470547492978688)). Open a `Contributors` section in your README to publicly thanks them (ex: [vuepress contributors section](https://github.com/vuejs/vuepress#code-contributors)).
+> Be nice with people that helped you! Some open-source projects like [gatsby](https://github.com/gatsbyjs/gatsby) reward contributors with goodies. If you can't afford that, do a public post (on twitter or other platforms) about the contribution and mention the author ([here is an example of public thanks](https://twitter.com/FranckAbgrall/status/1139470547492978688)). Open a `Contributors` section in your README to publicly thanks them (ex: [vuepress contributors section](https://github.com/vuejs/vuepress#code-contributors)) or showcase them on your project documentation/website (ex: [Rythm.js demo page](https://okazari.github.io/Rythm.js/))
```
Comment (tbetous) [inline README.md]:
```suggestion
> Be nice with people that helped you! Some open-source projects like [gatsby](https://github.com/gatsbyjs/gatsby) reward contributors with goodies. If you can't afford that, do a public post (on twitter or other platforms) about the contribution and mention the author ([here is an example of public thanks](https://twitter.com/FranckAbgrall/status/1139470547492978688)). Open a `Contributors` section in your README to publicly thanks them or showcase them on your project documentation/website. Here some examples :
>
> - [vuepress (contributors README section)](https://github.com/vuejs/vuepress#code-contributors)
> - [Rythm.js (random highlighted contributor on demo page)](https://okazari.github.io/Rythm.js/)
```

File: README-fr.md [hunk 4]
```diff
@@ -209,7 +209,7 @@ Une langue est manquante ? Vous pensez qu'il est possible d'améliorer ce cheat
...
+> Certains projets open-source comme [gatsby](https://github.com/gatsbyjs/gatsby) récompensent leurs contributeurs avec des goodies. Si vous n'en n'avez pas les moyens, faites une publication (sur twitter ou d'autres plateformes) qui mentionne la contribution et son auteur afin de le remercier (ex: [Post sur twitter](https://twitter.com/FranckAbgrall/status/1139470547492978688)). Vous pouvez aussi ouvrir une section `Contributors` sur votre README afin d'afficher publiquement votre gratitude envers vos contributeurs (ex: [Section contributeurs de vuepress](https://github.com/vuejs/vuepress#code-contributors)) ou encore les mettre en avant sur le site/la documentation de votre projet (ex: [Page de démo de Rythm.js](https://okazari.github.io/Rythm.js/))
```
Comment (tbetous) [inline README-fr.md]:
```suggestion
> Certains projets open-source comme [gatsby](https://github.com/gatsbyjs/gatsby) récompensent leurs contributeurs avec des goodies. Si vous n'en n'avez pas les moyens, faites une publication (sur twitter ou d'autres plateformes) qui mentionne la contribution et son auteur afin de le remercier (ex: [Post sur twitter](https://twitter.com/FranckAbgrall/status/1139470547492978688)). Vous pouvez aussi ouvrir une section `Contributors` sur votre README afin d'afficher publiquement votre gratitude envers vos contributeurs ou encore les mettre en avant sur le site/la documentation de votre projet.
>
> - [vuepress (section contributeurs dans le README)](https://github.com/vuejs/vuepress#code-contributors)
> - [Rythm.js (Mise en avant de manière aléatoire d'un contributeur sur la page de démo)](https://okazari.github.io/Rythm.js/)
```

**Comment by tbetous** [comment]
@Okazari , I am not sure that a link to a picture is the best experience we can give to readers. I would like to give to readers a true example of a website he can visit with a simple click. I am looking for a board exactly like conference all or botpress  but easily reachable (no login / no install)

**Comment by Okazari** [comment]
@tbetous You're right, unfortunatly i don't have any easily reachable board like this in my mind :thinking:

**Pull request merged**